src/com/oltpbenchmark/api/BenchmarkModule.java

```diff
@@ -155,11 +158,15 @@ public final Connection makeConnection() throws SQLException {
         }
 
         int r = dataSourceCounter.getAndIncrement() % workConf.getNodes().size();
-
-        String connectStr = String.format("jdbc:postgresql://%s:%d/%s",
-                                          workConf.getNodes().get(r),
-                                          workConf.getPort(),
-                                          workConf.getDBName());
+        String connectStr = null;
+        if (workConf.getJdbcURL().length()>0) {
+            connectStr=workConf.getJdbcURL();
+        } else {
+            connectStr = String.format("jdbc:postgresql://%s:%d/%s",
+                                            workConf.getNodes().get(r),
+                                            workConf.getPort(),
+                                            workConf.getDBName());
```

Comment:
nit: alignment

Reply:
I didn't touch this, but tried to fix it anyway.